internal/core/services/rules.go

```diff
@@ -8,55 +8,47 @@ import (
 	log "github.com/sirupsen/logrus"
 
 	"github.com/Knetic/govaluate"
-	"github.com/iancoleman/strcase"
 )
 
-var ruleFunctions = map[string]govaluate.ExpressionFunction{
-	"Get": func(args ...interface{}) (interface{}, error) {
-		if len(args) != 2 {
-			return nil, errors.New("Get takes 2 args")
-		}
-		arg1 := args[0]
-		arg2 := args[1]
+var errNoValueFound = errors.New("no value was found in the map with the key")
 
-		labelMap, ok := arg1.(map[string]string)
-		if !ok {
-			return nil, errors.New("label must be a map[string]string")
-		}
-		label, ok := arg2.(string)
-		if !ok {
-			return nil, errors.New("label must be of type string")
-		}
+// get returns the value of the specified key in the supplied map
+func get(args ...interface{}) (interface{}, error) {
+	if len(args) != 2 {
+		return nil, errors.New("Get takes 2 args")
+	}
+	arg1 := args[0]
```

Comment:
I know you didn't write this originally, but can you make these name (`arg1` and `arg2`) more descriptive.  The current names are basically useless.